build/sections/hero.html

```diff
@@ -0,0 +1,42 @@
+<div class="container relative pt-5 xl:min-h-[500px]">
+    <div class="mb-8 xl:mb-0 xl:absolute xl:top-[37%] xl:transform xl:-translate-y-1/2">
+        <p class="text-center xl:text-left text-2xl md:text-[50px] text-white font-bold font-raleway">
+            <span>Win-Win:</span> <br>
+            Europas <span class="text-primary-green inline-block md:my-5">Digitalisierung</span> <br>
+            profitiert von Afrikas Talenten 
+        </p>
+        <p class="text-center xl:text-left mt-3 text-white">Erweiteren Sie Ihre Entwicklungsteams mit Ojemba</p>
+    </div>
+    <img class="mx-auto xl:absolute xl:bottom-0 xl:right-0" width="650" src="Images/Ojembees.png" alt="">
+</div>
+
+
+<div class="container w-full relative py-3 bg-hero bg-cover bg-no-repeat flex flex-col gap-y-8 md:flex-row md:justify-between gap-x-12 lg:gap-x-32 -mt-[68px] text-white">
+    <div class="flex items-center gap-x-4">
+        <div>
+            <img width="120" class="max-w-[60px]" src="Images/hand-coins.png" alt="">
+        </div>
+        <div>
+            <h1 class="text-primary-green">Kostengünstig</h1>
+            <p class="md:max-w-[200px]">niedriges Lohnkostenniveau im europäischen Vergleich</p>
+        </div>
```

Comment:
Looks good, but you forgot to use the fonts they provided in Figma, you may check and apply them where needed.